src/main/scala/org/labrad/manager/Listener.scala

```diff
@@ -12,42 +14,65 @@ import scala.concurrent.ExecutionContext
 /**
```

Comment:
I haven't been consistent with that, but I think 1 blank line is consistent with the style guides for java (anything we can do to make scala look less foreign is probably good :) ). Do you have a preference either way?